src/components/TopBar/HamburgerMenu.jsx

```diff
@@ -59,14 +62,25 @@ const HamburgerMenu = createMenu({
         );
 
         if (isExperimental) {
-          items.push(
-            <MenuItem
-              key="exportRepo"
-              onClick={isRepoExportInProgress ? noop : onExportRepo}
-            >
-              {t('top-bar.export-repo')}
-            </MenuItem>,
-          );
+          if (hasExportedRepo) {
+            items.push(
+              <MenuItem
+                key="updateRepo"
+                onClick={isRepoUpdateInProgress ? noop : onUpdateRepo}
```

Comment:
Seems like we could use `isDisabled` here? Will get some styling for free and also it will handle the `noop` on click.

Reply:
sgtm - also updated gist and classroom export so we're consistent across the burger

Reply:
if there’s one thing i like in a burger, it’s cheese. if there’s two, it’s cheese and consistency.